playbooks/kubevirt.yml

```diff
@@ -0,0 +1,291 @@
+---
+- hosts: all
+  become: true
+  vars:
+    selinux_state: permissive
+    kubernetes_version: 1.13.5
+    pod_network: 10.244.0.0/16
+    flannel_version: a70459be0084506e4ec919aa1c114638878db11b
+    network_operator: 0.7.0
+    kubevirt_version: v0.16.3
+    ovs_cni_version: master
+  roles:
+    - selinux
+    - epel_repositories
+  tasks:
+    - name: disable swap
+      command: swapoff -a
+
+    - name: deconfigure swap
+      mount:
+        src: /swapfile
+        fstype: swap
+        path: none
+        state: absent
+
+    - name: modprobe br_netfilter
+      modprobe:
+        name: br_netfilter
+
+    - name: configure bridge iptables
+      sysctl:
+        name: "{{ item }}"
+        value: 1
+        sysctl_file: /etc/sysctl.d/k8s.conf
+      with_items:
+        - net.bridge.bridge-nf-call-ip6tables
+        - net.bridge.bridge-nf-call-iptables
+
+    - name: install needed network manager libs
+      yum:
+        name:
+        - NetworkManager-glib
+        - NetworkManager
+
+    - name: Configure bridge
+      nmcli:
+        state: present
+        type: bridge
+        conn_name: foreman
```

Comment:
@evgeni while this creates the _foreman_ bridge on the kubevirt node, it seems to miss a part of network configuration: The goal is to enable "boot from network" from a VM running on kubevirt via a PXE running on foreman node. The chart below shows the two vagrant vms (I created only the first one on my env, haven't run the foreman vm).

```
kubevirt                                            foreman/foreman-proxy/pxe
-----------------                                   -----------------
|               |                                   |               |
|          eth0-+------<default>--------------------+-eth0          |
|               |                                   |               |
|               |                                   |               |
| foreman--eth1-+------<foremankubevirt>------------+-eth1---PXE    |
|    |          |                                   |               |
|    vm         |                                   |               |
-----------------                                   -----------------
```
There are couple of questions:
1. foreman bridge isn't attached to any interface per kubevirt's yml if I understand the *nmcli* part correctly. Also wasn't able to see it is connected to any interface:
```
[vagrant@centos7-infra-kubevirt ~]$ nmcli con show
NAME         UUID                                  TYPE      DEVICE  
cni0         0aaf09b4-ba6e-4dbf-b999-c16e5e01a3ce  bridge    cni0    
docker0      22cafa5d-86bc-4501-b5f1-d28723c08a9d  bridge    docker0 
System eth0  5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03  ethernet  eth0    
System eth1  9c92fad9-6ecb-3e6c-eb4d-8a47c6f50c04  ethernet  eth1 
```

But was able to see it was created:
```
[vagrant@centos7-infra-kubevirt ~]$  nmcli -f bridge con show foreman
bridge.mac-address:                     --
bridge.stp:                             yes
bridge.priority:                        128
bridge.forward-delay:                   15
bridge.hello-time:                      2
bridge.max-age:                         20
bridge.ageing-time:                     300
bridge.group-forward-mask:              0
bridge.multicast-snooping:              yes
```
2. Which network should be used for connecting the vms via foreman bridge via host's interface to foreman's pxe ?
Will we use eth1 as in the chart ? via foremankubevirt libvirt network ?
3. What are the changes needed to be applied (if any) on foreman's vm to make it operable as the chart shows ?